atd-vze/src/views/Crashes/Crash.js

```diff
@@ -71,7 +80,7 @@ function Crash(props) {
   const [formData, setFormData] = useState({});
   const [isEditingCoords, setIsEditingCoords] = useState(false);
 
-  const { getRoles } = useAuth0();
+  const { getRoles, user } = useAuth0();
```

Comment:
Thanks to @mddilley for the suggestion to check the email as stored in the session state, not local storage, which can be easily modified by a user.